configs/nvme-storage-configurator/configmap.yaml

```diff
@@ -0,0 +1,72 @@
+apiVersion: v1
+kind: ConfigMap
```

Comment:
lets use kustomize configmap generator for generating this file:
1. then we can edit each script/unit file in its own file and have the benefit of IDEs.
2. when the content of the configmap changes, it will cause a roll out of the daemonset that will retrigger the `configure*` scripts.

Reply:
now using kustomize to generate the configmap